src/plugins/felixhayashi/tiddlymap/js/graph/ViewAbstraction.js

```diff
@@ -284,24 +274,18 @@ class ViewAbstraction {
    * @param {string} [name] - Instead of all configurations being returned,
    *     only the configuration named name is returned. The initial "config."
    *     may be omitted.
-   * @param {boolean} [isRebuild] - True if to rebuild the cache, false otherwise.
    * @result {string|Object} If `type` is not specified an object containing
    *     all configurations is returned, otherwise a single value will be returned.
    */
-  getConfig(name, isRebuild) {
-
-    let config;
+  getConfig(name) {
 
-    if (!isRebuild && this.config) {
+    const self = this;
+    const config = $tw.wiki.getCacheForTiddler(this.configTRef, "tmap-config", function() {
```

Comment:
please use an arrow function and remove `self = this` above. the arrow function automatically binds to `this`